File: src/chap03/section1/VarargsTest.java
package chap03.section1;

public class VarargsTest {

    public static void main(String[] args) {
        varargsParam(1, 2, 3, 4);
    }

    public static void varargsParam(int... ts){
        for(int t: ts){
            System.out.println(t);
        }
    }
}

File: src/chap03/section1/VarargsTest.kt
package chap03.section1

fun main() {

    normalVarargs(1, 2, 3, 4) // 4개의 인자 구성
    normalVarargs(4, 5, 6)    // 3개의 인자 구성

}

fun normalVarargs(vararg counts: Int): Unit {
    for (num in counts) {
        print("$num ")
    }
    print("\n")
}